cmd/kubeadm/app/componentconfigs/kubelet_test.go

```diff
@@ -125,60 +124,17 @@ func TestKubeletDefault(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "Service subnet, explicitly disabled dual stack defaulting works",
-			clusterCfg: kubeadmapi.ClusterConfiguration{
-				FeatureGates: map[string]bool{
-					features.IPv6DualStack: false,
-				},
-				Networking: kubeadmapi.Networking{
-					ServiceSubnet: "192.168.0.0/16",
-				},
-			},
-			expected: kubeletConfig{
-				config: kubeletconfig.KubeletConfiguration{
-					FeatureGates: map[string]bool{
-						features.IPv6DualStack: false,
-					},
-					StaticPodPath: kubeadmapiv1.DefaultManifestsDir,
-					ClusterDNS:    []string{"192.168.0.10"},
-					Authentication: kubeletconfig.KubeletAuthentication{
-						X509: kubeletconfig.KubeletX509Authentication{
-							ClientCAFile: constants.CACertName,
-						},
-						Anonymous: kubeletconfig.KubeletAnonymousAuthentication{
-							Enabled: utilpointer.BoolPtr(kubeletAuthenticationAnonymousEnabled),
-						},
-						Webhook: kubeletconfig.KubeletWebhookAuthentication{
-							Enabled: utilpointer.BoolPtr(kubeletAuthenticationWebhookEnabled),
-						},
-					},
-					Authorization: kubeletconfig.KubeletAuthorization{
-						Mode: kubeletconfig.KubeletAuthorizationModeWebhook,
-					},
-					HealthzBindAddress: kubeletHealthzBindAddress,
-					HealthzPort:        utilpointer.Int32Ptr(constants.KubeletHealthzPort),
-					RotateCertificates: kubeletRotateCertificates,
-					ResolverConfig:     resolverConfig,
-					CgroupDriver:       constants.CgroupDriverSystemd,
-				},
-			},
-		},
 		{
 			name: "Service subnet, enabled dual stack defaulting works",
 			clusterCfg: kubeadmapi.ClusterConfiguration{
-				FeatureGates: map[string]bool{
-					features.IPv6DualStack: true,
-				},
+				FeatureGates: map[string]bool{},
 				Networking: kubeadmapi.Networking{
 					ServiceSubnet: "192.168.0.0/16",
 				},
 			},
 			expected: kubeletConfig{
 				config: kubeletconfig.KubeletConfiguration{
-					FeatureGates: map[string]bool{
-						features.IPv6DualStack: true,
-					},
+					FeatureGates:  map[string]bool{},
```

Comment:
[1]

Reply:
The expected one is by default with this.
The UT would fail if it is removed here.